doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitsche_mixer_with_pbt_impeller.rst

```diff
@@ -180,54 +254,85 @@ It is often desirable to obtain the time-averaged velocity field when simulating
 
-Linear solver parameters
+Linear Solver Parameters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Relatively standard parameters are used for the linear solver. From our experience, the AMG preconditioner is more robust with the Nitsche IB than the traditional ILU.
+Relatively standard parameters are used for the :doc:`../../../parameters/cfd/linear_solver_control`. From our experience, the AMG preconditioner is more robust with the Nitsche immersed boundaries than the traditional ILU.
 
 .. code-block:: text
 
   subsection linear solver
     set method                    = amg
     set max iters                 = 200
-    set relative residual         = 1e-3
     set minimum residual          = 1e-7
-    set amg preconditioner ilu fill                  = 0
     set amg preconditioner ilu absolute tolerance    = 1e-8
     set amg preconditioner ilu relative tolerance    = 2.00
     set amg aggregation threshold = 1e-10  
-    set amg smoother sweeps       = 2      
-    set amg smoother overlap      = 1      
-    set verbosity                 = verbose
-    set max krylov vectors      = 200
+    set max krylov vectors        = 200
   end
 
+:raw-html:`<br />`
+
 Running the simulation
 ----------------------
-Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``gls_nitsche_navier_stokes_33`` executable is within your path, the simulation can be launched by typing:
+Launching the simulation is as simple as specifying the executable name and the parameter file. For this more complex example, it is highly advised to use ``mpirun`` to parallelize the computation, with ``<np>`` the number of processes, defined accordingly with your machine's number of cpu. The simulation can be launched by typing, in a terminal opened in the example folder:
 
 .. code-block:: text
 
-  gls_nitsche_navier_stokes_33 mixer.prm
+  mpirun -np <np> </path/to/the/build/applications/gls_nitsche_navier_stokes_33/gls_nitsche_navier_stokes_33 mixer.prm>
 
-Lethe generates a larger number of files when the Nitsche immersed boundary is used. The ``mixer.pvd`` contains the 3D visualization of the velocity, pressure, and auxiliary variables such as the average velocity. Additional files will also be generated, notably the ``mixer_solid_triangulation_00.pvd`` and ``mixer_solid_particles_00.pvd``, corresponding to the Nitsche IB mesh and the discrete particles inserted at the Gauss points of the solid triangulation, respectively. The solid particles enable the Nitsche restriction visualization, while the solid triangulation is used for animation purposes.
+Lethe generates a larger number of files when the Nitsche immersed boundary is used:
+* the ``mixer.pvd`` contains the 3D visualization of the velocity, pressure, and auxiliary variables such as the average velocity ;
```

Comment:
The bullets does not appear on the html. I think it needs an indentation?

Reply:
Well spotted! I need to add a line between the text and bullet point